x-pack/plugins/alerts/server/task_runner/task_runner.ts

```diff
@@ -317,12 +317,14 @@ export class TaskRunner {
             previousStartedAt,
           };
         },
-        (err: Error) => {
+        // @ts-expect-error
+        async (err: Error) => {
```

Comment:
Did this pass typecheck? I'd expect it not to as `map` is a Result type and wouldn't be able to handle a Promise being returned.

Reply:
nope it definitely did not haha. Just pushed an update. Thanks.